docs/outlook/onmessagefromchanged-onappointmentfromchanged-events.md

```diff
@@ -275,10 +271,15 @@ Event handlers must be configured for the `OnNewMessageCompose` and `OnMessageFr
 
 1. From the **./src/commands** folder, open **commands.html**.
 
-1. Replace the existing **script** tag with the following code.
+1. If you'll run the add-in in Outlook on Mac, replace the existing **script** tag with the following code. Otherwise, skip to the next step.
```

Comment:
```suggestion
1. To run the add-in in Outlook on Mac, replace the existing **script** tag with the following code. Otherwise, skip to the next step.
```

Reply:
I was having trouble committing your suggestion directly, so I manually made the change.